apps/app-article-server/src/browser/components/article-content.js

```diff
@@ -251,6 +259,25 @@ class Content extends Component {
     );
   }
 
+  handleInternalLinks(htmlString) {
+    const validHostnames = ['www.hbl.fi', 'www.ostnyland.fi', 'www.vastranyland.fi']
+
+    const $ = cheerio.load(htmlString, null, false)
+    const $links = $('a')
+    $links.each((i, el) => {
+      const $el = $(el)
+      const urlObj = new URL($el.attr('href'))
+
+      if(validHostnames.includes(urlObj.hostname) && urlObj.pathname.startsWith('/artikel/')) {
```

Comment:
Are the intra-article links always to www.hbl.fi and never to hbl.fi? The validHostnames could include the non-www alternatives to be sure.

Reply:
Will probably be only to www, but yeah I don't think it hurts to add them just in case.